app/views/modules/_footer.html.erb

```diff
@@ -13,16 +13,55 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-   <div class="container-fluid">
-	  <footer>
-	   <ul>
-	   	 <li><%= image_tag "AvalonMediaSystem_Logo_A.png", alt: "Avalon Media System - #{t(:release_label)} #{Avalon::VERSION}" %>
-	     <li><a href="http://www.avalonmediasystem.org">Avalon Media System Project Website</a> </li>
-       <li><%= link_to "Contact Us", comments_path %></li>
-	   </ul>
-	   <div class="release">
-	   	<%=release_text%>
-	   	<%= git_commit_info(" (%s %s [%s])") if Rails.env == 'development' %>
-	   </div>
-       </footer>
+
+<div class="footer dark row-fluid">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-4">
+        <h2>Quick Links</h2>
+        <ul>
+          <li><a href="/users/sign_in">Login</a></li>
+          <li><a href="/catalog?q=&search_field=all_fields&utf8=%E2%9C%93">Browse</a></li>
+        </ul>
+      </div>
+      <div class="col-sm-4">
+        <h2 class="hidden-xs">Policies</h2>
+        <ul>
+          <li><a href="/about_eraav">About ERA A<span class="plus">&plus;</span>V</a></li>
+          <li><a href="/policies">Policies</a></li>
+          <li><a href="/technology">Technology and Partnerships</a></li>
+        </ul>
+      </div>
+      <div class="col-sm-4">
+        <h2 class="hidden-xs">Get Help</h2>
+        <ul>
+          <li><a href="/contact">Contact Us</a></li>
+          <li><a href="/deposit">How to Deposit</a></li>
+        </ul>
+      </div>
+    </div>
+    <div class="row copyright">
+      <a href="https://library.ualberta.ca"> University of Alberta Libraries</a> |
+      University of Alberta, Edmonton, AB, Canada T6G 2R3 |
+      <a href="https://library.ualberta.ca/about-us/policies/privacy-policy">
+        Web Site Privacy Policy
+      </a>
+    </div>
+    <div class="row">
+      <a href="http://www.avalonmediasystem.org"><%= t(:powered_by_avalon) %></a>
+      <!-- Avalon Media System Release <%= Avalon::VERSION %> -->
     </div>
+  </div>
+
+  <% if Rails.env.production? %>
+  <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    ga('create', 'UA-77935097-1', 'auto');
```

Comment:
This can't be hardcoded like this, needs to be put into secrets.yml or something similar.

(any dev/test/staging/etc environments will sending data to google analytics, heavily skewing your metrics for what you think is production)